src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/HmppsAuthMockServer.kt

```diff
@@ -210,4 +210,171 @@ class HmppsAuthMockServer : WireMockServer(WIREMOCK_PORT) {
         ),
     )
   }
+
+  fun stubUserDefaultSearchNoResults() {
+    stubFor(
+      get("/auth/api/user/search")
+        .willReturn(
+          aResponse()
+            .withStatus(HttpStatus.OK.value())
+            .withHeaders(HttpHeaders(HttpHeader("Content-Type", "application/json")))
+            .withBody(
+              """
+               {
+                  "content": [],
+                  "pageable": {
+                      "sort": {
+                          "empty": false,
+                          "sorted": true,
+                          "unsorted": false
+                      },
+                      "offset": 0,
+                      "pageSize": 10,
+                      "pageNumber": 0,
+                      "paged": true,
+                      "unpaged": false
+                  },
+                  "totalPages": 0,
+                  "totalElements": 0,
+                  "last": true,
+                  "size": 10,
+                  "number": 0,
+                  "sort": {
+                      "empty": false,
+                      "sorted": true,
+                      "unsorted": false
+                  },
+                  "first": true,
+                  "numberOfElements": 0,
+                  "empty": true
+              }
+              """.trimIndent(),
+            ),
+        ),
+    )
+  }
+
+  fun stubUserSearchAllFiltersWithResults(name: String) {
```

Comment:
Should this all filters stub not include status and auth source too?

Reply:
yes - will update